src/hgvs/variantmapper.py

```diff
@@ -491,12 +519,15 @@ def _replace_reference(self, var):
             # this is an out-of-bounds variant
             return var
 
-        seq = self.hdp.get_seq(var.ac, seq_start, seq_end)
+        seq = self.hdp.get_seq(ac, seq_start, seq_end)
 
         if len(seq) != seq_end - seq_start:
             # tried to read beyond seq end; this is an out-of-bounds variant
             return var
 
+        if _type == "g" and mapper and mapper.strand == -1:
```

Comment:
Interesting, just out of curiosity, when can the strand be negative for a g.? Inversions?

Reply:
I'm not really sure. I don't have a simple, science-y explanation for why this is the case. My fist pass at this PR didn't include it and we noticed some mappings gave the reverse compliments of the expected values. After some more digging we found that only g type variants on the negative strand as described by the alignment mapper were affected.